pebblo/entity_classifier/utils/regex_pattern.py

```diff
@@ -5,12 +5,12 @@
 """
 
 regex_secrets_patterns = {
-    "github-token": r"""\b((?:ghp|gho|ghu|ghs|ghr)_[a-zA-Z0-9]{36,255})\b""",
-    "slack-token": r"""(xoxb|xoxp|xapp|xoxa|xoxr)\-[0-9]{10,13}\-[a-zA-Z0-9\-]*""",
+    "github-token": r"""((?:ghp|gho|ghu|ghs|ghr|github_pat)_[a-zA-Z0-9_]{36,255})""",
```

Comment:
**False Positives**: In addition to matching existing token patterns, the new `github-token` regex will also match new patterns that include underscores, which could lead to false positives. 

For example, it might incorrectly match strings like `ghp_1234567890abcdef1234567890abcdef1234_extra` or `ghp_12345_67890_abcdef_12345_67890_abcdef`, which may not be valid tokens.